python-template/{{cookiecutter.placeholder_repo_name}}/README.rst

```diff
@@ -31,7 +33,7 @@ One Time Setup
 .. code-block::
 
   # Clone the repository
-  git clone git@github.com:edx/{{ cookiecutter.repo_name }}.git
+  git clone git@github.com:openedx/{{ cookiecutter.repo_name }}.git
```

Comment:
There are a bunch of other edx org references in the badge URLs at the bottom if you wanted to fix those now, or it can wait for https://github.com/openedx/edx-cookiecutters/issues/217

Reply:
I've updated them.